apis/ec2/src/main/java/org/jclouds/ec2/compute/domain/EC2HardwareBuilder.java

```diff
@@ -429,6 +444,89 @@ public static EC2HardwareBuilder t2_large() {
             .processors(ImmutableList.of(new Processor(1.0, 0.4))).rootDeviceType(RootDeviceType.EBS);
    }
 
+   /**
+    * @see InstanceType#T2_XLARGE
+    */
+   public static EC2HardwareBuilder t2_xlarge() {
+      return new EC2HardwareBuilder(InstanceType.T2_XLARGE).t2()
+              .ram(16384)
+              .processors(ImmutableList.of(new Processor(2.0, 0.4))).rootDeviceType(RootDeviceType.EBS);
```

Comment:
Also t2 number of processors seem to be wrong, t2.xlarge has 4vCPU

Reply:
https://aws.amazon.com/it/ec2/instance-types/t2/

I wasn't 100% sure on my values, but i used the existing `t2.large` and `t2.xlarge` as my example space.  From the above link I'm seeing `t2.large(vCpu=2)` and `t2.xlarge(vCpu=4)`

Should I change all of the t2 processors to match the product detail page?

Reply:
After looking at the M4 for additional sanity, I agree with what you are proposing and will make the change soon.

Reply:
Code should match what the AWS documentation and the javadoc, yes.

t2.large is ok but t2.xlarge should be 4 yes